src/Tasks/Microsoft.NET.Build.Tasks/ResolvePublishAssemblies.cs

```diff
@@ -42,6 +44,12 @@ public ITaskItem[] AssembliesToPublish
         {
             get { return _assembliesToPublish.ToArray(); }
         }
+        [Output]
+        public ITaskItem[] PackagesPublished
+        {
+            get { return _assembliesToPublish.ToArray(); }
```

Comment:
I'm not sure I understand why this has a 2nd Output parameter that is the exact same value as `AssembliesToPublish`.  Why can't we just have 1 output parameter?